src/CheckoutSdk/Sessions/Source/SessionSource.cs

```diff
@@ -0,0 +1,38 @@
+﻿using Checkout.Common;
+using Newtonsoft.Json;
+
+namespace Checkout.Sessions.Source
```

Comment:
JsonProperty members are not required.

Reply:
I had to add the jsonProperty for all protected members.